book/src/bridge/traits.md

```diff
@@ -19,8 +19,8 @@ impl UniquePtr<A> {} // explicit CXX trait implementation of UniquePtr for A
 impl cxx_qt::Trait for A {} // explicit CXX-Qt trait implementation of Trait for A
 ```
 
-- [CxxQtType](./cxxqttype.md) - trait to reach the Rust implementation of a QObject
-- [Constructor](./constructor.md) - custom constructor
-- [Initialize](./initialize.md) - execute Rust code when the object is constructed
-- [Locking](./locking.md) - marker trait whether locking is enabled
-- [Threading](./threading.md) - marker trait whether CXX-Qt threading should be enabled
+- [CxxQtType](https://docs.rs/cxx-qt/latest/cxx_qt/trait.CxxQtType.html) - trait to reach the Rust implementation of a QObject
```

Comment:
we may need to add more docs to the trait itself ?

Reply:
Yes, though not necessarily for `CxxQtType`, as I think it's a less important trait.

 I think in general we're removing too much information here.
We should copy some of the text from `traits/` into their code documentation.

Reply:
I think `CxxQtType` is still important to mention, as it's not obvious to new people how do you go from the C++ context to the Rust context.